product-scenarios/2-Bridging-systems-that-communicate-in-different-protocols/2.1-HTTP to HTTP protocol translation/2.1.1-Receive an HTTP request from a client and deliver it over HTTP message to the downstream without altering the content/src/test/java/org/wso2/carbon/esb/scenario/test/HttpMessageWithoutAlteringDownstreamTest.java

```diff
@@ -0,0 +1,51 @@
+package org.wso2.carbon.esb.scenario.test;
+
+import org.apache.http.HttpResponse;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.wso2.carbon.esb.scenario.test.common.HttpConstants;
+import org.wso2.carbon.esb.scenario.test.common.ScenarioConstants;
+import org.wso2.carbon.esb.scenario.test.common.ScenarioTestBase;
+import org.wso2.esb.integration.common.utils.clients.SimpleHttpClient;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+// Test http message to the downstream without altering the content
+public class HttpMessageWithoutAlteringDownstreamTest extends ScenarioTestBase{
+    private final String carFileName = "2_1_1_7_httpToHttpProtocolTranslationCompositeApplication_1.0.0";
+    private String proxyServiceUrl;
+
+    @BeforeClass(alwaysRun = true)
+    public void init() throws Exception {
+        super.init();
+        proxyServiceUrl = getProxyServiceURLHttp("2_1_1_7_httpToHttpProtocolTranslationProxy\n");
+        deployCarbonApplication(carFileName);
+    }
+
+    @Test(description = "2.1.1.7 Testcase", enabled = true, dataProvider = "2.1.1.7")
```

Comment:
Please put only the id for the description? word 'Testcase' is redundant. This entry is used for the traceability of test results and testcases in testcase.md

Reply:
Fixed.